submissions/kt-std/memory-pair-game/js/script.js

```diff
@@ -0,0 +1,145 @@
+const container = document.querySelector(".container"),
+	numCards = 9,
+	cardsData = ((assetsPath) => {
+		const arr = [];
```

Comment:
`arr` is a very generic name. Let's find a name that describes what this variable does.

Reply:
@AMashoshyna lots of thanks for your review especially during holidays season! I am so thankful!
I've updated the variable name with another one to clearly express what it actually does.